### ISSUE TITLE
Googlemaps not working when build from source in Mac OS X

### ISSUE THREAD

**Issue by SeppoTakalo** [subsurface/subsurface#588]
Using Qt 5.9.1.
I cloned the master repository and followed instructions in [INSTALL](https://github.com/Subsurface-divelog/subsurface/blob/master/INSTALL)

Built using the `./subsurface/script/build.sh` script which builds succesfully but I noticed following warning:
```
cp: cannot stat '/Users/st/Qt/5.9.1/clang_64/plugins/geoservices/libqtgeoservices_googlemaps.dylib': No such file or directory
```

Then when starting the application:
```
st@st-laptop:~/src/subsurface $ ./subsurface/build/Subsurface.app/Contents/MacOS/Subsurface 
qml: MapWidget.qml: cannot find a plugin named: googlemaps
qrc:/MapWidget.qml:21: Error: Cannot assign [undefined] to QDeclarativeGeoMapType*
MapWidget ready
```

And in the UI, the map widget shows miniature map icon on top of white background.
![nayttokuva 2017-09-20 kello 20 08 30](https://user-images.githubusercontent.com/3104794/30657272-a8f38ff8-9e3f-11e7-9fe4-1b3b892ccd4a.png)

**Comment by dirkhh** [comment]
Did you happen to save a log of the build? It should have cloned the sources, built and installed that googlemaps plugin...

**Comment by SeppoTakalo** [comment]
Install-root folder seem to have directory tree replicated that probably should have been targeted to `/`

```
st@st-laptop:~/src/subsurface $ ls -R install-root/Users/
install-root/Users/:
st

install-root/Users/st:
Qt

install-root/Users/st/Qt:
5.9.1

install-root/Users/st/Qt/5.9.1:
clang_64

install-root/Users/st/Qt/5.9.1/clang_64:
lib  plugins

install-root/Users/st/Qt/5.9.1/clang_64/lib:
cmake

install-root/Users/st/Qt/5.9.1/clang_64/lib/cmake:
Qt5Location

install-root/Users/st/Qt/5.9.1/clang_64/lib/cmake/Qt5Location:
Qt5Location_QGeoServiceProviderFactoryGooglemaps.cmake

install-root/Users/st/Qt/5.9.1/clang_64/plugins:
geoservices

install-root/Users/st/Qt/5.9.1/clang_64/plugins/geoservices:
libqtgeoservices_googlemaps.dylib       libqtgeoservices_googlemaps_debug.dylib
...
```

`/Users/st` is my home dir.

**Comment by SeppoTakalo** [comment]
I fixed the problem manually by going into this `src/googlemaps/build` directory and called `make install`

Then I restarted the build script and everything seems to work OK. Also it fixed the freezing issue #589

**Comment by dirkhh** [comment]
I still want to fix the actual build issue - it's good that you fixed it for yourself, but the script should try to get this right.

**Comment by mfbernardes** [comment]
Seems fixed by 0620c81594
